src/query/algorithms/triangles/StreamingTriangles.cpp

```diff
@@ -96,29 +93,21 @@ std::string StreamingTriangles::countCentralStoreStreamingTriangles(std::string
 
 std::map<long, std::unordered_set<long>> StreamingTriangles::getCentralAdjacencyList(unsigned int graphID,
                                                                                      unsigned int partitionID) {
-    std::map<long, std::unordered_set<long>> adjacencyList;
+    unsigned long maxLabel = std::stol(Utils::getJasmineGraphProperty("org.jasminegraph.nativestore.max.label.size"));
+    GraphConfig gc{maxLabel, graphID, partitionID, "app"};
+    std::unique_ptr<NodeManager> nodeManager(new NodeManager(gc));
 
-    GraphConfig gc;
-    gc.graphID = graphID;
-    gc.partitionID = partitionID;
-    gc.maxLabelSize = std::stoi(Utils::getJasmineGraphProperty("org.jasminegraph.nativestore.max.label.size"));
-    gc.openMode = "app";
-    NodeManager* nm = new NodeManager(gc);
-    adjacencyList = nm->getAdjacencyList(false);
-    return adjacencyList;
+    return nodeManager->getAdjacencyList(false);
 }
 
 std::vector<std::pair<long, long>> StreamingTriangles::getEdges(unsigned int graphID, unsigned int partitionID,
                                                                 long previousCentralRelationCount) {
     std::vector<std::pair<long, long>> edges;
-    GraphConfig gc;
-    gc.graphID = graphID;
-    gc.partitionID = partitionID;
-    gc.maxLabelSize = std::stoi(Utils::getJasmineGraphProperty("org.jasminegraph.nativestore.max.label.size"));
-    gc.openMode = "app";
-    NodeManager* nodeManager = new NodeManager(gc);
-
-    std::string dbPrefix = nodeManager->getDbPrefix();
+    unsigned long maxLabel = std::stol(Utils::getJasmineGraphProperty("org.jasminegraph.nativestore.max.label.size"));
```

Comment:
Have you committed this `org.jasminegraph.nativestore.max.label.size` property into JasmineGraph properties file?

Reply:
yes

Reply:
Ok